recipes/minizip-ng/all/test_package/CMakeLists.txt

```
@@ -4,5 +4,5 @@ project(test_package LANGUAGES C)
 find_package(minizip REQUIRED CONFIG)
 
 add_executable(${PROJECT_NAME} test_package.c)
-target_link_libraries(${PROJECT_NAME} PRIVATE MINIZIP::minizip)
+target_link_libraries(${PROJECT_NAME} PRIVATE minizip::minizip)
```

```suggestion
target_link_libraries(${PROJECT_NAME} PRIVATE MINIZIP::minizip)
```
Please, revert it. Use the official CMake target provided by the upstream only.